repl/src/main/scala_2.10/org/apache/spark/repl/h2o/H2OInterpreter.scala

```diff
@@ -101,4 +101,24 @@ class H2OInterpreter(sparkContext: SparkContext, sessionId: Int) extends BaseH2O
 
 object H2OInterpreter {
   def classOutputDirectory = H2OIMain.classOutputDirectory
+
+
+  def getUserJars(conf: SparkConf): Seq[String] = {
```

Comment:
WHy this method is duplicated by a method below?

Reply:
It differs in few lines because of different APIs if Scala 2.10 and Scala 2.11